scalableminds/webknossos
# Allow to activate a tree group

### URL of deployed dev instance (used for testing):
- https://activetreegroup.webknossos.xyz

### Steps to test:
- test usual tree functionality
- create groups
- activate groups and rename them (via the text box)
- try some actions while a group is active
  - create node should show a toast
  - create tree should create a tree in the active group

### Issues:
- fixes #2871 

------
